tools/bin/3d-tiles-tools.js

```diff
@@ -197,3 +253,61 @@ function readGlbWriteB3dm(inputPath, force, argv) {
                 });
         });
 }
+
+function readB3dmWriteGlb(inputPath, outputPath, force) {
+    outputPath = defaultValue(outputPath, inputPath.slice(0, inputPath.length - 4) + 'glb');
+    return fileExists(outputPath)
+        .then(function(exists) {
+            if (!force && exists) {
+                console.log('File ' + outputPath + ' already exists. Specify -f or --force to overwrite existing file.');
+                return;
+            }
+            return fsReadFile(inputPath)
+                .then(function(data) {
+                    return fsWriteFile(outputPath, extractB3dm(data).glb);
+                });
+        });
+}
+
+function readAndOptimizeB3dm(inputPath, outputPath, force) {
+    var options = {};
+    if (defined(optionArgs)) {
+        // Specify input for argument parsing even though it won't be used
+        optionArgs.push('-i');
+        optionArgs.push('null');
+        options = GltfPipeline.parseArguments(optionArgs);
+    }
+    outputPath = defaultValue(outputPath, inputPath);
```

Comment:
Instead of overwriting if no output path is given, it should add a suffix to the filename, like `batchedWithBatchTableBinary-optimized.b3dm`